### PR TITLE
Update CSV fields

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
This patch adds better description to NFD_Operator CSV and updates kustomize values so `make bundle` doesn't remove labels and annotations required by build system

Also remove a non needed API variable about configMap status, since we only track the CRD status not obj lvl status

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Now the install process looks like this 
image

**Comment by ArangoGutierrez** [comment]
Now when installed, the Operator also provides some helpful documentation 
image

File: config/manifests/bases/node-feature-discovery-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ metadata:
     support: Red Hat
   labels:
     operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.arm64: supported
```
Comment (dagrayvid) [inline config/manifests/bases/node-feature-discovery-operator.clusterserviceversion.yaml:42]: Duplicate lines?

**Comment by dagrayvid** [comment]
The changes all look good. I think you should explain the API change in the PR description

**Review by ArangoGutierrez — COMMENTED** [review]
> The changes all look good. I think you should explain the API change in the PR description



> The changes all look good. I think you should explain the API change in the PR description

Thanks, done!

**Pull request merged**